platform/src/abstractions/tests/Platform.Sql.Tests/Builders/PlatformQueryTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿using Platform.Sql.QueryBuilders;
+using Xunit;
+
+namespace Platform.Sql.Tests.Builders;
+
+public class PlatformQueryTests
+{
+    [Fact]
+    public void ShouldAddUrnEqualParameter()
+    {
+        const string expected = "SELECT * FROM Mock WHERE URN = @URN\n";
```

Comment:
nitpick: Lots of repetition of the mock `SELECT` from `MockPlatformQuery`. Is there any way this can be refactored away?